### DISCUSSION TITLE
Not found undotree command

### DISCUSSION THREAD

**Discussion by Xiaoyang-Lyu** [NvChad/NvChad#3000] [Q&A] [answered]
Hi everyone, I’m new to Neovim and NVChad. I added the line `“mbbill/undotree"` to plugins/init, and when I ran `:Lazy` it indicated that undotree was loaded. However, I can't use any of the undotree commands in the editor. How should I troubleshoot or debug this?

**Comment by Xiaoyang-Lyu** [comment]
Correction: In the `:Lazy` command, undotree wasn't loaded, but it has been installed.

**Comment by Xiaoyang-Lyu** [comment, marked as answer]
Sorry everyone, the issue is resolved. I needed to manually run `:Lazy load` Maybe I should set `lazy = false` in the configuration?

**Comment by siduck** [comment]
@Xiaoyang-Lyu  lazy load it , `cmd = "Undotree"`  ( write the correct command )
